firefox-ios/Client/Application/SceneDelegate.swift

```diff
@@ -96,16 +96,16 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     ) {
         guard let url = URLContexts.first?.url,
               let route = routeBuilder.makeRoute(url: url) else { return }
-        sceneCoordinator?.findAndHandle(route: route)
-
         sessionManager.launchSessionProvider.openedFromExternalSource = true
+        sceneCoordinator?.findAndHandle(route: route)
```

Comment:
`openedFromExternalSource` was set after the `findAndHandle` was called. I see this as a low risk change, in case this has anything to do with some issues we have.